tools/aws_lab_setup/sample-vars.yml

```diff
@@ -0,0 +1,15 @@
+ec2_region: us-east-1                 # region where the nodes will live
+ec2_az: us-east-1a                    # availability zone
+ec2_name_prefix: SEANTEST1113         # name prefix for all the VMs
+admin_password: ansible
+## Optional Variables
+#ec2_vpc_id: vpc-1234aaaa              # EC2 VPC ID in your region, if not supplied will be automatically created
+#ec2_vpc_subnet_id: subnet-5678bbbb    # EC2 subnet ID in your VPC, if not supplied will be automatically created
+#ec2_key_name: your_key                # SSH key in AWS to put in all the instances, if not supplied will be automatically created
+email: no                             # Set this if you wish to disable email
+#sendgrid_user: username               # username for the Sendgrid module.  Not required if "email: no" is set
+#sendgrid_pass: 'passwordgoeshere'     # sendgrid accound password.  Not required if "email: no" is set
+#sendgrid_api_key: 'APIkey'            # Instead of username and password, you may use an API key. Don't define both. Not required if "email: no" is set
+#instructor_email: 'Ansible Instructor <helloworld@acme.com>'  # address emails arrive from. Not required if "email: no"
+#networking: yes                       # Set this if you want the workshop in networking mode (NOT USED YET)
+localsecurity: false                   # skips firewalld installation and SE Linux when turned to false
```

Comment:
I honestly haven't looked into what it would take to make them more secure, but I'm not a fan of turning off SELinux during customer demos, as it just promotes the idea that SELinux is still hard. Can we make the default as it turned on, and still allow it to be turned off via this variable?

Reply:
thats exactly how it works, the default is on, this is just a sample vars where we turn it off

Reply:
Derp. Sorry I see that now. I misunderstood. Disregard!